src/stactools/sentinel2/__init__.py

```diff
@@ -11,4 +11,4 @@ def register_plugin(registry):
     registry.register_subcommand(commands.create_sentinel2_command)
 
 
-__version__ = "0.4.2"
+__version__ = "0.5.0-pre1"
```

Comment:
Python (/ PyPI) wants formats like this:

> X.YaN   # Alpha release
> X.YbN   # Beta release
> X.YrcN  # Release Candidate
> X.Y     # Final release

Reference: https://peps.python.org/pep-0440/#pre-releases